internal/config/schema.go

```diff
@@ -32,11 +32,32 @@ type FolderUploadJob struct {
 	// Album is the album where objects will be uploaded.
 	// If the Album option is not set, the objects will not be associated with an album in Google Photos.
 	//
-	// These are the valid values: "name:" and "auto:".
+	// These are the valid values: "name:", "auto:", "template".
 	//   "name:" : Followed by the album name in Google Photos (album names are not unique, so the first to match
 	//             will be selected)
 	//   "auto:" : Followed either "folderPath" or "folderName" will use an autogenerated album name based on the
 	//             object's folder path or object's folder name.
+	//   "template": Followed by a template string that can contain the following predefine tokens and functions:
+	//              Tokens:
+	//                 %_folderpath% - full path of the folder containing the file.
+	//                 %_directory% - name of the folder containing the file.
+	//                 %_parent_directory% - Replaced with the name of the parent folder of the file.
+	//                 %_day% - day of the month the file was created (in "DD" format).
+	//                 %_month% -month the file was created (in "MM" format).
+	//				   %_year% - year the file was created (in "YYYY" format).
```

Comment:
Do you mind to align these two lines for readability?

Reply:
No looks like mixed spaces and tabs and Git hub show tabs as 4. while my setting is showing tabs at 2. Changing to all spaces